src/js/components/Table/Paginated/PaginatedTable.tsx

```diff
@@ -102,20 +100,48 @@ const Filter = (
             : convertObjsToValueLabel(options as { [key: string]: string | React.ReactNode })
         }
       />
-    ) : (
+    );
+  };
+};
+
+const DateFilterType = (
```

Comment:
Small nitpick. We probably dont want to call this a `Type` since its quite a bit more than a type. I think `DateFilter` is probably fine. Thoughts?